SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+## Supported Versions
+
+| Version     | Supported          | Status                | Comments                             |
+| ----------- | ------------------ | --------------------- | ------------------------------------ |
+| 1.7.x       | :white_check_mark: | Long Term Support     | See [Long Term Support policy][0]    |
```

Comment:
1.7 will no longer be supported when 1.8.0 is getting released which should happen soon.

Reply:
Agreed, but we should probably update this at that time?

Reply:
I’ll leave it to @petebacondarwin, I just wanted to make sure we won’t miss this.

Reply:
It might be easy to miss if this PR & #17027 land independently.